autogen/oai/client.py

```diff
@@ -233,6 +519,18 @@ def yes_or_no_filter(context, response):
         if ERROR:
             raise ERROR
         last = len(self._clients) - 1
+        if len(self._clients) == 0:
+            raise RuntimeError(
+                "No model client is active. Please populate the config list or register any custom model clients."
+            )
+        # Check if all configs in config list are activated
+        non_activated = [
```

Comment:
`not_activated = any(isinstance(client, PlaceHolderClient) for client in self._clients`.

Reply:
I want it to print the names of the client classes that are missing for a more informative error message, the above will only respond true/false